### PR TITLE
LPS-152450 Add instance configuration to set default value for auto tagging on update

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp [hunk 1]
```diff
@@ -558,6 +558,7 @@ renderResponse.setTitle(headerTitle);
 								id='<%= liferayPortletResponse.getNamespace() + "updateAutoTags" %>'
 								label='<%= LanguageUtil.get(request, "update-auto-tags") %>'
 								name='<%= liferayPortletResponse.getNamespace() + "updateAutoTags" %>'
+								value="<%= dlAdminDisplayContext.isUpdateAutoTags() %>"
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp:561]: boolean cannot be converted to String

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp (hunk 1 above)
Comment (adolfopa) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp:561]: :facepalm:

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**